lib/cerner/oauth1a/access_token.rb

```diff
@@ -22,7 +22,9 @@ class AccessToken
       def self.from_authorization_header(value)
         params = Protocol.parse_authorization_header(value)
 
-        raise OAuthError.new('', nil, 'version_rejected') unless params[:oauth_version]&.eql?('1.0')
+        unless params[:oauth_version].nil? || params[:oauth_version]&.eql?('1.0')
```

Comment:
I prefer to have complex boolean checks use 'if' and only do 'unless' when it's just a single boolean.

I think that would mean this code could be elided to be:
```
raise OAuthError.new('', nil, 'version_rejected') if params[:oauth_version] && !params[:oauth_version].eql?('1.0')
```

Or is that line too long?

Reply:
It's 3 characters too long. Could make it multiple lines and still use an 'if', but it will still make the MethodLength too long by 1 line.

```ruby
if params[:oauth_version] && !params[:oauth_version].eql?('1.0')
  raise OAuthError.new('', nil, 'version_rejected')
end
```

Reply:
Converted 'unless' into 'if': https://github.com/cerner/cerner-oauth1a/pull/5/commits/31f56397b5e76add30320dcafa03823e2ff2efc5#diff-6dcb2be10e4906ed8d1053b9caa4fe12L25